draft-ietf-trans-gossip.md

```diff
@@ -268,15 +302,22 @@ learn about other sites visited by the HTTPS client. Second, auditors
 and monitors receiving SCTs from the HTTPS server would learn
 information about other HTTPS servers visited by its clients.
 
-When the client later connects to an HTTPS server it again receives a 
-set of SCTs. The client MUST add to its store those SCTs that are issued by known logs
-and associate them with the exact domain name contacted, as described above. The client MUST
-send to the server SCTs in the store that are associated with that
-domain name unless they were received from that server in the current TLS session.
-
-\[TODO: fix the above paragraph -- it is vague and confusing. maybe
-  an example including a client caching at most one SCT per host+log
-  would clarify\]
+When the client later connects to the HTTPS server it again receives a 
+set of SCTs and calculates a certificate chain, and again creates a 
+sct_feedback or similar object. If this object does not exactly match 
+an existing object in the store, then the client MUST add this new 
```

Comment:
No... Line 285 is about discarding SCTs that aren't signed by a known log, and this MUST (line 308) is about adding the constructed object to the store...